### ISSUE TITLE
Softmax probabilities instead of logits

### ISSUE THREAD

**Issue by fugokidi** [fra31/auto-attack#102]
Thank you for the open source auto-attack.
I have a soft-voting ensemble which outputs the average of softmax probabilities.
Since softmax is not invertible, I cannot convert probabilities to logits.
Is it ok to provide softmax probabilities instead of logits to auto-attack?

**Comment by ScarlettChan** [comment]
您好，您的邮件已收到!

**Comment by fra31** [comment]
Hi,

using the softmax output should work fine. However, you might also want to have a look a some methods designed specifically for ensembles like [this](https://arxiv.org/abs/2011.14031).

Hope this helps!

**Comment by fugokidi** [comment]
@fra31 Thank you. It's helpful.
